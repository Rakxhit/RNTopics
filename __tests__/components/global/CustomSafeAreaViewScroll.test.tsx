import {Text} from 'react-native';
import CustomSafeAreaScrollView from '../../../src/components/global/CustomSafeAreaViewScroll';
import {render} from '@testing-library/react-native';

describe('CustomSafeAreaViewScroll component', () => {
  it('should render ', () => {
    const {getByText} = render(
      <CustomSafeAreaScrollView>
        <Text>First Text</Text>
      </CustomSafeAreaScrollView>,
    );
    expect(getByText('First Text')).toBeTruthy();
  });
  it('should render correctly', () => {
    const {toJSON} = render(
      <CustomSafeAreaScrollView>
        <Text>Test Text</Text>
      </CustomSafeAreaScrollView>,
    );
    expect(toJSON()).toMatchSnapshot();
  });
});
