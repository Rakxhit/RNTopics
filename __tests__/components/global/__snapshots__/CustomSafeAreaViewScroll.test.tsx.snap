// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomSafeAreaViewScroll component should render correctly 1`] = `
<RCTSafeAreaView
  style={
    [
      {
        "backgroundColor": "#1a1a1a",
        "flex": 1,
      },
      undefined,
    ]
  }
  testID="safe-area-view"
>
  <RCTScrollView
    contentContainerStyle={
      {
        "flex": 1,
        "padding": 10,
        "paddingTop": 20,
      }
    }
    nestedScrollEnabled={true}
  >
    <View>
      <Text>
        Test Text
      </Text>
    </View>
  </RCTScrollView>
</RCTSafeAreaView>
`;
